diff --git a/include/constants/species.h b/include/constants/species.h
--- a/include/constants/species.h
+++ b/include/constants/species.h
@@ -1606,6 +1606,9 @@ enum __attribute__((packed)) Species
 
     SPECIES_CUSTOM_START = SPECIES_GLIMMORA_MEGA,
     // Add any custom species between here and SPECIES_CUSTOM_END
+    SPECIES_CINDURSA,
+    SPECIES_PYROGRIZZ,
+    SPECIES_INFERURSA,
     SPECIES_CUSTOM_END,
 
     SPECIES_EGG = SPECIES_CUSTOM_END,
diff --git a/src/data/pokemon/species_info.h b/src/data/pokemon/species_info.h
--- a/src/data/pokemon/species_info.h
+++ b/src/data/pokemon/species_info.h
@@ -174,6 +174,8 @@ const struct SpeciesInfo gSpeciesInfo[] =
     #include "species_info/gen_8_families.h"
     #include "species_info/gen_9_families.h"
 
+    #include "species_info/pokeman_journey_custom.h"
+
     [SPECIES_EGG] =
     {
         .frontPic = gMonFrontPic_Egg,
